src/pages/api/preview.js

```diff
@@ -0,0 +1,36 @@
+import wp from "@/promisetracker/lib/wp";
+
+export default async function preview(req, res) {
+  const {
+    postType,
+    postId,
+    revisionId,
+    _wpnonce: nonce,
```

Comment:
Since `nonce` doesn't work, lets remove it here as well as [here](https://github.com/CodeForAfrica/HURUmap-Dashboard/blob/e21644d3040e972902fe3be418f2fd9153b9329c/wp-content/themes/hurumap/functions.php#L237)